dubbo-rpc/dubbo-rpc-rest/src/main/java/org/apache/dubbo/rpc/protocol/rest/request/RequestFacade.java

```diff
@@ -56,6 +58,17 @@ protected void initHeaders() {}
     protected void initParameters() {
         String requestURI = getRequestURI();
 
+        String decodedRequestURI = requestURI;
+        try {
+            decodedRequestURI = URLDecoder.decode(decodedRequestURI, "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            // do nothing
+        }
```

Comment:
This is hardcoded, we can set utf-8 by default, but it must come from `Accept-Charset` first.

Reply:
ok, I'll make a adjustment

Reply:
I've made an adjustment according to your opinon, please take a look again, take your time@